_data/heroku-landing-page-tabs-code.yml

```diff
@@ -0,0 +1,111 @@
+- id: fargate
+  title: Fargate
+  content: |
+    <figure><figcaption>ecs-fargate-service/main.tf</figcaption>
+      <pre>
+      <code class="language-js">
+        module "fargate_service" {
+          source = "git::git@github.com:gruntwork-io/terraform-aws-ecs.git//modules/ecs-service?ref=v1.0.8"
+
+          service_name    = var.service_name
+          ecs_cluster_arn = aws_ecs_cluster.fargate_cluster.arn
+
+          desired_number_of_tasks        = var.desired_number_of_tasks
+          ecs_task_container_definitions = data.template_file.container_definition.rendered
+          launch_type                    = "FARGATE"
+          platform_version               = "1.4.0"
+
+          # Network information is necessary for Fargate, as it required VPC type
+          ecs_task_definition_network_mode = "awsvpc"
+          ecs_service_network_configuration = {
+            subnets          = data.aws_subnet_ids.default.ids
+            security_groups  = [aws_security_group.ecs_task_security_group.id]
+            assign_public_ip = true
+          }
+
+          # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html#fargate-tasks-size.
+          # Specify memory in MB
+          task_cpu    = 256
+          task_memory = 512
+
+          enable_ecs_deployment_check      = var.enable_ecs_deployment_check
+          deployment_check_timeout_seconds = var.deployment_check_timeout_seconds
+
+          # EFS volumes
+          efs_volumes = local.efs_volumes
+
+          # Make sure all the ECS cluster resources are deployed before deploying any ECS service resources. This is also
+          # necessary to avoid issues on 'destroy'.
+          depends_on = [aws_ecs_cluster.fargate_cluster]
+        }
+      </code>
+      </pre>
+      <figure>
+- id: ecs
+  title: ECS
+  content: |
+    <figure><figcaption>ecs-cluster/main.tf</figcaption>
+      <pre>
+      <code class="language-js">
+        module "ecs_cluster" {
+          source = "git::git@github.com:gruntwork-io/terraform-aws-ecs.git//modules/ecs-cluster?ref=v1.0.0"
```

Comment:
Same here. We should be using the Service Catalog. Below is a rough snippet you can use.

```hcl
module "ecs_service" {
  source = "git::git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/services/ecs-service?ref=v0.51.0"  

  service_name          = "my-heroku-app"
  launch_type           = "EC2"
  ecs_cluster_arn       = module.ecs_cluster.arn 
  
  desired_number_of_tasks = 3

  container_definitions = [
    {
      name         = "my-heroku-app"
      image        = "https://123456789012.dkr.ecr.us-east-1.amazonaws.com/my-heroku-app:v1"
      essential    = true
      cpu          = 256
      memory       = 512
      portMappings = [
        {
          "hostPort"      = 3000
          "containerPort" = 3000
          "protocol"      = "tcp"
        }
      ]      
    }
  ]

  domain_name = "example.your-domain.com"
}

module "ecs_cluster" {
  source = "git::git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/services/ecs-cluster?ref=v0.51.0"  

  cluster_name = "my-ecs-cluster"

  cluster_min_size      = 3
  cluster_max_size      = 10
  cluster_instance_type = "m4.large"
  cluster_instance_ami  = "ami-abcd1234"
}
```